integrated_channels/blackboard/client.py

```diff
@@ -154,8 +154,37 @@ def delete_content_metadata(self, serialized_data):
 
     def create_assessment_reporting(self, user_id, payload):
         """
-        Not implemented yet
+        Post a learner's subsection assessment grade to the integrated Blackboard course.
+
+        Parameters:
+        -----------
+            user_id (str): The shared email between a user's edX account and Blackboard account
```

Comment:
interesting I thought we used the name user_email in another function.. is this better named user_email?

Reply:
https://github.com/edx/edx-enterprise/pull/1063#discussion_r533737416

Had this discussion with @taliaedX on the Canvas assessment work. The "ID" here will always be an email, but in the overall integrated channels, this is not email specific. We designate the `remote_user_id` to be whatever we want in the specific channel's transmitter. Because we pass `remote_user_id` into the client from the base transmitter, I decided to keep this as `user_id`

Reply:
makes sense, got it